concent_api/utils/logging.py

```diff
@@ -212,21 +259,11 @@ def log_file_status(task_id: str, subtask_id: str, requestor_public_key: bytes,
     ))
 
 
-def get_task_id_for_logging(message):
-    task_id = get_field_from_message(message, 'task_id')
-    if not isinstance(task_id, str):
-        task_id = ''
-    return task_id
-
-
-def get_subtask_id_for_logging(message):
-    subtask_id = get_field_from_message(message, 'subtask_id')
-    if not isinstance(subtask_id, str):
-        subtask_id = ''
-    return subtask_id
-
-
-def log_request_received(path_to_file, operation: FileTransferToken.Operation):
+def log_request_received(
```

Comment:
Woludn't you make here  a good use of `replace_element_to_unavailable_instead_of_none` decorator?

Reply:
Like before...